scapy/modules/nmap.py

```diff
@@ -25,18 +25,17 @@
 from scapy.arch import WINDOWS
 from scapy.error import warning
 from scapy.layers.inet import IP, TCP, UDP, ICMP, UDPerror, IPerror
-from scapy.packet import NoPayload
+from scapy.packet import NoPayload, Packet
 from scapy.sendrecv import sr
-from scapy.compat import plain_str, raw
+from scapy.compat import plain_str, raw, Dict, List, Tuple, Optional, cast, Union
+from scapy.plist import SndRcvList, PacketList
 import scapy.libs.six as six
 
-
```

Comment:
Please avoid unneeded change to preserve the code history as much as possible.

Reply:
ok